app/views/content/become-a-further-education-teacher.md

```diff
@@ -19,18 +19,16 @@ keywords:
   - apprencticeship
 ---
 
-Further education is any study that takes place after secondary school but outside of higher education (e.g. university).
+If you’re interested in teaching young people and adults over 16 who aren’t studying for a degree, then further education might be for you. It covers three key areas: 
```

Comment:
I'm wondering if this page might read a bit better if we split it up with a few h2s? e.g. we could add a 'what is further education?' h2 before the 'it covers:' line?